docs/user-guide/topics/metrics.md

```diff
@@ -41,3 +42,15 @@ Iter8 defines `counter` and `gauge` metric types which are analogous to the corr
     
     Gauges are typically used for measured values like temperatures or current memory usage, but also "counts" that can go up and down, like the number of concurrent requests.
 
+???+ note "Histogram"
+<!-- TODO -->
+
+???+ note "Sample"
+<!-- TODO -->
+
+???+ note "Summary"
+<!-- TODO -->
```

Comment:
I still need some help filling out this section

Reply:
Lets handle in the next PR.